### PR TITLE
fix: display notification when asset deployment fails

### LINKED ISSUE #11338: Minting - User not informed of failure to mint on Arbitrum Goerli
# Bug Report

## Description

During minting, after signing the transaction in the final step the process failed. The Pokt error banner was displayed but the user was not informed that the minting did not take place. They were returned to the minting welcome screen with no other info or toast.

## Steps to reproduce

Attempt to mint a token on using Arb testnet.
Sign transaction
User is returned to the minting welcome screen, the pokt error banner is displayed temporarily, no toast or other information is provided.

Console

```

ERR 2023-06-29 16:59:02.613+01:00 Error deploying collectibles               topics="community-tokens-service" tid=5732788 file=service.nim:336 message="\nstatus-go error [methodName:collectibles_deploy, code:-32000, 
message:fallback failed with \'intrinsic gas too low\'. run error was \'intrinsic gas too low\' ]\n"
```

#### Expected behavior
Minting screen with error state for the collectible and toast.

### Additional Information

- Status desktop version: 0.13RC5
- Operating System: Mac

### PR DESCRIPTION
**Pull request by 0x-r4bbit**
Closes #11338

### CODE REVIEW

**Review by jrainville — COMMENTED** [review]
Looks good. Do we also have the `Minting screen with error state` as stated in the issue?

**Comment by 0x-r4bbit** [comment]
Nope, this just adds the notification.
Will add the changes for the error screen as well, once I've figured out what they are.

**Comment by 0x-r4bbit** [comment]
I realized the error state handling **does** exist in the view. I've updated the PR to make use of the already existing `SIGNAL_COMMUNITY_TOKEN_DEPLOY_STATUS_CHANGED` event, which causes the view to update accordingly.

File: ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml [hunk 1]
```diff
@@ -644,7 +644,7 @@ StatusSectionLayout {
                 return
             }
 
-            Global.displayToastMessage(title, qsTr("View on etherscan"), "",
+            Global.displayToastMessage(title, url === "" ? qsTr("Something went wrong") : qsTr("View on etherscan"), "",
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml:647]: I'd be more descriptive here, like "Error deploying assets"? (not sure about the actual message)

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml (hunk 1 above)
Comment (0x-r4bbit) [reply ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml:647]: Originally I wanted to expose the underlying error message that we get when the transaction fails, but unfortunately we can't easily access it with our current backend implementation so I went with something generic.

I can update this to something you've mentioned, chances are low that there'd be any other semantic error when `url == ""`

Comment (caybro) [reply ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml:647]: Yeah, not a big deal

**Review by endulab — APPROVED** [review]

**Pull request merged**